settings.php

```diff
@@ -71,6 +71,18 @@
             get_string('showrefresh_desc', 'local_envbar', null, true),
             true));
 
+    // Show time to next refresh.
+    $presentation->add(new admin_setting_configcheckbox('local_envbar/shownextrefresh',
```

Comment:
do we need this? why not just only show it if nextrefresh is non empty?

Reply:
fixed